src/components/BookList.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Book from './Book';
+
+const BookList = () => (
+
+  <ul>
+    <Book />
+  </ul>
+
+);
```

Comment:
- [ ] As you need to use `props` in the `Book.js`. So you need to pass the data from here now.  Kindly fix this issue. Here I am giving an example:
`<Book title="The Hunger Games" author="Suzane Collins" category="Action" /> `
`<Book title="The Knight of Valour" author="Hollywood" category="Animation" /> `